### ISSUE TITLE
Highlighting fails after inline conditional statement

### ISSUE THREAD

**Issue by joaodforce** [natewallace/angular2-inline#43]
As the following image shows, if a inline conditional statement is set inside a parameter, the highlight fails for the next until the next double quotes I think.

<img width="528" alt="image" src="https://user-images.githubusercontent.com/9410916/45779618-dbb08c00-bc31-11e8-8991-e8c4509d1582.png">

**Comment by 0x054** [comment]
@joaodforce I posted this in the open pull request but wanted to surface my finding here as well.

For me, the highlighting continues to work if you wrap the ternary in `()`.  Does it continue to work for you?

i.e:

```
<div [className]="(secondaryButton ? 'col-md-3' : 'col-md-9')"></div>
```

**Comment by joaodforce** [comment]
@IVIyg0t Yes, this does indeed fix the highlighting and also doesn't break the code, I've learned this workaround after some time I've posted this.

**Comment by 0x054** [comment]
Awesome!

**Comment by ruilop** [comment]
@joaodforce the () ternary wrap worked for me, but Prettier is configured to remove them, so it would be aswesome if we could avoid them, also because they aren't really needed...

**Comment by nejsimon** [comment]
Indeed prettier removes the parentheses because they´re unnecessary causing the highlighting to look broken again, so unfortunately it's not a viable workaround.
